libs/gltfio/include/gltfio/SimpleViewer.h

```diff
@@ -125,6 +131,8 @@ class SimpleViewer {
     bool mShowWireframe = false;
     bool mEnableSunlight = true;
     bool mEnableDithering = true;
+    bool mEnablePrepass = true;
+    bool mEnableFxaa = true;
```

Comment:
You should also add a toggle for MSAA 4x

Reply:
Done